msteams-platform/bots/what-are-bots.md

```diff
@@ -13,26 +13,30 @@ The GIF below shows a user conversing with a bot in a one-to-one chat using both
 
 ![FAQ Plus gif](~/assets/images/FAQPlusEndUser.gif)
 
-## How bots work
+## Build  a bot for Teams with the Microsoft Bot Framework
+
+The Microsoft Bot Framework](https://dev.botframework.com/) is a rich SDK for building bots using C#, Java, Python, and JavaScript. If you already have a bot that's based on the Bot Framework, you can easily adapt it to work in Microsoft Teams. We recommend you use either C# or Node.js to take advantage of our [SDKs](/microsoftteams/platform/#pivot=sdk-tools). These packages extend the basic Bot Builder SDK classes and methods as follows:
+
+* Use specialized card types like the Office 365 Connector card.
```

Comment:
Can we change this to Adaptive cards? or Bot framework cards? O365 cards going to move into legacy soon